python/ribasim_testmodels/ribasim_testmodels/time.py

```diff
@@ -102,3 +102,157 @@ def flow_boundary_time_model():
     )
 
     return model
+
+
+def conditions_on_discrete_flow_model():
```

Comment:
Could you add a docstring on what this model exactly represents (and tests)?

(As far as I can tell, it always listens to the flow of node 1, checks the flow, and as different values are exceeded, the pump is given a higher rate?)

Reply:
I'll ad a docstring (and give it a better name). What the model does is explained in https://github.com/Deltares/Ribasim/issues/447, and demonstrated above. We discussed this at length a few weeks back, about going back to the setpoint once the level is outside an accepted range.